src/flowr/internal-session.ts

```diff
@@ -110,6 +113,30 @@ export class FlowrInternalSession implements FlowrSession {
 		return dataflowGraphToMermaid(result.dataflow, result.normalize.idMap)
 	}
 
+	async retrieveAstMermaid(document: vscode.TextDocument): Promise<string> {
+		if(!this.shell) {
+			return ''
+		}
+		const result = await new SteppingSlicer({
```

Comment:
We should schedule a migration issue for this, once df2 is released as 2.0.0 of flowr

Reply:
#84